include/bios_handler.hpp

```diff
@@ -88,6 +88,13 @@ class IbmBiosHandler : public BiosHandlerInterface
      * @param[in] i_fcoInBios - FCO value.
      */
     void saveFcoToVpd(int64_t i_fcoInBios);
+
+    /**
+     * @brief API to save AMM data into VPD.
+     *
```

Comment:
can you describe this API in few sentences like what is AMM and why it is saved in VPD. so that anyone who looks into it should understand by reading your doxygen.

Reply:
Doxygen explanation is for function usability. 
It is not for educating people about the functionality.

Reply:
This is just a BIOS attribute saved in the VPD , so no point explaining  what it is . It does not even make much sense to many on how it works.  So IMO don't think we need an explanation , this is similar to write keyword value  for me

Reply:
then why we need doxygen here? why can't we simply have apis only? 

IMO, doxygen is to document the API, based on which one can understand the usecase of the API. adding a brief with abbreviations like AMM doesn't add any value to the reader.

Reply:
>. Doxygen explanation is for function usability.

As explained above, It provides some basic info about what the function is about and how it should be used.
Like: 
what parameters needs to be sent to call this function.
what return type can be expected.
will the api needs exception handling or not etc.

It is not to explain the technicalities of the function, for that some other documentation should be used.
else why you want only for AMM why not VPD? How do you know that everyone who will come across this API's doxygen will understand what VPD is?